contents/blog/the-state-of-plugins.md

```diff
@@ -0,0 +1,81 @@
+---
+date: 2021-11-28
+title: The state of plugins on PostHog
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+Author: yakko-majuri
+featuredImage: ../images/blog/running-content.png
+featuredImageType: full
+---
+
+The plugin server has become an integral part of PostHog, as it is responsible for event ingestion. However, it isn’t called “ingestion server” because it was originally designed to run plugins. So, how are those going?
+
+I used a combination of PostHog, Metabase, Python scripts, and Google Sheets to compile, analyze, and display the data you see here to answer that question as transparently as possible. 
+
+## TL;DR
+
+- Most teams using PostHog use plugins (if GeoIP is included)
+- Relatively few Cloud teams use plugins (if GeoIP is excluded)
+- Self-hosted users are way more likely to use plugins than Cloud users
+- Export plugins are key for our larger customers (particularly self-hosted ones)
+- Simple plugins covering basic functionality that PostHog lacks are quite popular
+- The plugins ecosystem still needs a lot of work 
+
+## Why we like plugins
+
+Fun fact: the average length for the top six most popular plugins on PostHog is only 57 lines of code. Some, such as the Timestamp Parser, are only 16 lines long! The other thing they have in common is that they cover basic functionality which we haven’t built natively into PostHog yet. 
+
+This is a deliberate decision. It’s true that we could spend engineering time to build these features, such as filtering nested properties, but plugins enable us to quickly solve the same challenge without the resource overhead. Then, once we fully understand the challenge, we can create a better solution — some plugins are literally the duct tape version of full fledged features we’re building at the moment. 
+
+Plugins also enable us to build useful functionality in a matter of hours. The Twitter Followers plugin hits an API and creates an event once a day. It would never make sense to build this natively, but it took just 30 minutes to make it as a plugin which is used by 30+ users.
```

Comment:
```suggestion
Plugins also enable us to build useful functionality in a matter of hours. The Twitter Followers plugin hits an API and creates an event once a day. It would never make sense to build this natively, but it took just 30 minutes to make it as a plugin which is used by 30+ teams.
```

This was copied off my text, I know. But it is indeed 30 teams instead of users, and that sounds better